docker/Dockerfile.decanlp

```diff
@@ -0,0 +1,54 @@
+# override this to "stanfordoval/decanlp:latest-cuda" if cuda is desired
+ARG BASE_IMAGE=stanfordoval/decanlp:latest
```

Comment:
Is the dockerfile for stanfordoval/decanlp defined in another PR?

Reply:
It's defined [in the decanlp repo](https://github.com/stanford-oval/decaNLP/blob/master/dockerfiles/Dockerfile). In fact, it is already public on [dockerhub](https://hub.docker.com/r/stanfordoval/decanlp)

Reply:
I see . The image is quite large already.  If the size becomes an issue, we might consider caching embedding it on S3.

Reply:
Yeah but I want to reduce the use of `aws` and also avoid having to configure aws credentials just to read the embeddings from somewhere. Also, the image is layered, and the embeddings are early in the layers, so those layers should stay cached basically always while the code changes. This is basically the main advantage compared to downloading on demand every time the container is started (which happens for every single training job).

Reply:
An alternative design is to download on the demand from decanlp, but instead of saving in pod-local storage, use a volume mapped to a host (node) directory, which then would semi-persistent and shared among multiple pods on the same node. It would have the same space sharing behavior as the image layering, with more operational flexibility.

Reply:
I'm more worrying about the case when image pull results in network timeout due the large size.  The problem exacerbates when there are multiple nodes starting up at the same time.  It may not be an issue for now.  Let's worry about that when it happens.